lib/instrumentation/modules/tedious.js

```diff
@@ -14,7 +14,7 @@ var { getDBDestination } = require('../context')
 
 module.exports = function (tedious, agent, { version, enabled }) {
   if (!enabled) return tedious
-  if (!semver.satisfies(version, '^1.9.0 || 2.x || 3.x || ^4.0.1 || 5.x || 6.x || 7.x || 8.x || 9.x || 10.x || 11.x || 12.x || 13.x || 14.x || 15.x')) {
+  if (!semver.satisfies(version, '^1.9.0 || 2.x || 3.x || ^4.0.1 || 5.x || 6.x || 7.x || 8.x || 9.x || 10.x || 11.x || 12.x || 13.x || 14.x || 15.x || 16.x')) {
```

Comment:
Looks good. 

nit: We *could* consider simplifying this to `>=1.9.0 <17` (and just ignore that version 4.0.0, from 2018, was a bad release).

Reply:
Agree on simplification  but maybe also have an early return for `4.0.0`?

Reply:
> maybe also have an early return for 4.0.0?

Yup, totally.